webapp/src/wsclient.ts

```diff
@@ -299,6 +301,17 @@ class WSClient {
         }
     }
 
+    pluginStatusesChangedHandler(data: any): void {
+        if (this.pluginId === '') {
+            return
+        }
+
+        if (data.plugin_statuses.some((s: any) => s.plugin_id === this.pluginId)) {
+            Utils.log('Boards plugin has been updated, reloading...')
+            location.reload()
```

Comment:
Can this cause issues when the user has unsaved progress? For example, what would happen if the user was editing a card description or comment?

Also, would it makes sense to show a warning first and reload after some time, let's say, a minute? This can be addressed in a separate ticket.

Reply:
This was addressed as a quick solution, I agree that something like Chats' "There is a new version of Mattermost, click here to reload" would be safer in case a user is working with the tool when the update happens

Reply:
I have the same concern here. If this is a temporary solution, we should include a TODO comment here, and a ticket for resolving that sooner than later.

Reply:
I'm actually taking a stab at putting together a banner and a more long-term fix, will update with the new approach

Reply:
Done, PTAL 64f3db0